queries.sql

```diff
@@ -16,3 +16,66 @@ SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.13 ;
+
+
+-- Query and Update animals table
+
+BEGIN;
+
+UPDATE animals SET species='unspecified';
+
+ROLLBACK;
+
+
+BEGIN;
+
+UPDATE animals
+SET species ='digimon'
+WHERE name like '%mon';
+
+
+UPDATE animals
+SET species ='pokemon'
+WHERE name NOT like '%mon';
+
+COMMIT;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+
+BEGIN;
+
+SAVEPOINT SP1;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP2;
+
+UPDATE animals SET weight_kg=weight_kg * (-1);
+
+ROLLBACK TO SP1;
+
+UPDATE animals SET weight_kg=weight_kg * (-1) WHERE weight_kg < -1;
```

Comment:
- [ ] Excellent work implementing queries! Regarding this one, kindly remember to multiply only the weight_kg where animals are `negative` it's include -1 🥇 Kindly fix this one